docker/pom.xml

```diff
@@ -0,0 +1,18 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.nuxeo.ecm.platform</groupId>
+    <artifactId>nuxeo-explorer-packages-parent</artifactId>
+    <version>20.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>nuxeo-explorer-docker-parent</artifactId>
+  <packaging>pom</packaging>
+  <name>Explorer Docker Images</name>
+
+  <modules>
+    <module>explorer-previewer-docker</module>
```

Comment:
Maybe simply `explorer-previewer` since we're already in a `docker` directory?

Reply:
actually will probably rename it "explorer-docker" or "explorer" as the "previewer" notion is not needed here

Reply:
renamed it nuxeo-explorer-docker for consistency for now, i can change it to a better name once the rest is ok

Reply:
please tell if you find that's alright this way or if i should change it after all

Reply:
keep as is for now